Lib/test/test__xxsubinterpreters.py

```diff
@@ -759,7 +759,11 @@ def test_still_running(self):
         main, = interpreters.list_all()
         interp = interpreters.create()
         with _running(interp):
-            with self.assertRaises(RuntimeError):
+            self.assertTrue(interpreters.is_running(interp),
+                msg=f"Interp {interp} should be running before destruction.")
+
+            with self.assertRaises(RuntimeError,
+                    msg=f"Should not be able to destroy interp {interp} while it's still running."):
```

Comment:
Likewise with lining up "arg lines":

```suggestion
                                   msg=f"Should not be able to destroy interp {interp} while it's still running."):
```

Also, personally I prefer not to split the opening clause of a compound statement over multiple lines.  However, in this case it isn't so big a deal, though I'd still try to keep the colon as visually distinct as possible, like this:

```suggestion
                                   msg=f"Should not be able to destroy interp {interp} while it's still running."
                                   ):
```

However, I don't think that buys much.  I'll leave it to your discretion. :smile:

Reply:
> Also, personally I prefer not to split the opening clause of a compound statement over multiple lines. However, in this case it isn't so big a deal, though I'd still try to keep the colon as visually distinct as possible, like this:

I added the first two suggestions, but left out the `):` suggestion mainly because there's been some recent controversy about whether it's PEP8 compliant: https://discuss.python.org/t/pep-8-clarify-if-multiline-argument-list-with-a-closing-on-a-separate-line-is-acceptable/2948/10.